### PR TITLE
Docs: add new release artifact to readme.

### PR DESCRIPTION
**Pull request by syhily**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -192,25 +175,25 @@ If you have already built a JAR package with dependencies using the above shade
...
- --addclasspath pulsar-flink-connector-{{SCALA_BINARY_VERSION}}-{{PULSAR_FLINK_VERSION}}.jar
+ --addclasspath pulsar-flink-connector_{{SCALA_BINARY_VERSION}}-{{PULSAR_FLINK_VERSION}}.jar
 ```
 
 For more information on submitting applications through the CLI, see [Command-Line Interface](https://ci.apache.org/projects/flink/flink-docs-release-1.12/deployment/cli.html) .
 
 ## SQL client
 
-The [SQL Client](https://ci.apache.org/projects/flink/flink-docs-release-1.12/dev/table/sqlClient.html) is used to write SQL queries for manipulating data in Pulsar, you can use the `-addclasspath` option to add `pulsar-flink-connector-{{SCALA_BINARY_VERSION}}-{{PULSAR_FLINK_VERSION}}.jar` package.
+The [SQL Client](https://ci.apache.org/projects/flink/flink-docs-release-1.12/dev/table/sqlClient.html) is used to write SQL queries for manipulating data in Pulsar, you can use the `-addclasspath` option to add `pulsar-flink-connector_{{SCALA_BINARY_VERSION}}-{{PULSAR_FLINK_VERSION}}.jar` package.
```
Comment (jianyun8023) [inline README.md:191]: In our current release, there is no such package.

File: doc/README_CN.md [hunk 2]
```diff
@@ -193,19 +179,19 @@ Pulsar Flink 连接器用来从 Pulsar 中读取数据或者将结果写入 Puls
 
 ```
 ./bin/start-scala-shell.sh remote <hostname> <portnumber>
-  --addclasspath pulsar-flink-connector-{{SCALA_BINARY_VERSION}}-{{PULSAR_FLINK_VERSION}}.jar
+  --addclasspath pulsar-flink-connector_{{SCALA_BINARY_VERSION}}-{{PULSAR_FLINK_VERSION}}.jar
```
Comment (jianyun8023) [inline doc/README_CN.md:182]: In our current release, there is no such package.

File: doc/README_CN.md [hunk 3]
```diff
@@ -193,19 +179,19 @@ Pulsar Flink 连接器用来从 Pulsar 中读取数据或者将结果写入 Puls
...
 ## SQL 客户端
 
-如需使用 [SQL 客户端](https://ci.apache.org/projects/flink/flink-docs-release-1.12/dev/table/sqlClient.html)并编写 SQL 查询、操作 Pulsar 中的数据，用户可以使用 `--addclasspath` 参数直接添加 `pulsar-flink-connector-{{SCALA_BINARY_VERSION}}-{{PULSAR_FLINK_VERSION}}.jar`。
+如需使用 [SQL 客户端](https://ci.apache.org/projects/flink/flink-docs-release-1.12/dev/table/sqlClient.html)并编写 SQL 查询、操作 Pulsar 中的数据，用户可以使用 `--addclasspath` 参数直接添加 `pulsar-flink-connector_{{SCALA_BINARY_VERSION}}-{{PULSAR_FLINK_VERSION}}.jar`。
 
 **举例**
 
 ```
-./bin/sql-client.sh embedded --jar pulsar-flink-connector-{{SCALA_BINARY_VERSION}}-{{PULSAR_FLINK_VERSION}}.jar
+./bin/sql-client.sh embedded --jar pulsar-flink-connector_{{SCALA_BINARY_VERSION}}-{{PULSAR_FLINK_VERSION}}.jar
```
Comment (jianyun8023) [inline doc/README_CN.md:194]: Same as above.

File: README.md (hunk 1 above)
Comment (syhily) [reply README.md:191]: The package have been change to this name in this PR.

**Review by jianyun8023 — APPROVED** [review]

**Pull request merged**